packages/modules-sdk/src/medusa-module.ts

```diff
@@ -0,0 +1,108 @@
+import { asValue } from "awilix"
+import { createMedusaContainer } from "medusa-core-utils"
+import { moduleLoader, registerMedusaModule } from "./loaders"
+import {
+  ExternalModuleDeclaration,
+  InternalModuleDeclaration,
+  MODULE_RESOURCE_TYPE,
+  MODULE_SCOPE,
+} from "./types"
+import { loadModuleMigrations } from "./loaders/utils"
+
+const logger: any = {
+  log: (a) => console.log(a),
+  info: (a) => console.log(a),
+  warn: (a) => console.warn(a),
+  error: (a) => console.error(a),
+}
```

Comment:
This will be addresed in another PR and Logger will be moved to `medusa-core-utils`